stellarphot/tests/test_core.py

```diff
@@ -245,9 +245,9 @@ def test_photometry_data():
     assert phot_data.camera.read_noise == 10.0 * u.electron
     assert phot_data.camera.dark_current == 0.01 * u.electron / u.second
     assert phot_data.camera.pixel_scale == 0.563 * u.arcsec / u.pix
-    assert phot_data.observatory.lat.value == 46.86678
+    np.testing.assert_almost_equal(phot_data.observatory.lat.value, 46.86678)
```

Comment:
Cool, nice to know this function exists for checking floats.  Looked over the document, seems to be used fine here (I was concerned about precision, but default precision is 7 decimal places, which is clearly enough here).